catalog/Catalog.proto

```diff
@@ -57,6 +57,16 @@ message NUMAPlacementScheme {
   repeated BlockToNUMANodeEntry block_to_numa_node_map = 2;
 }
 
+message IndexScheme {
+  required uint32 num_indices = 1;
+
+  message IndexEntry {
+    required string index_name = 1;
+    repeated IndexSubBlockDescription index_description = 2;
+  }
+  repeated IndexEntry index_entries = 2;
```

Comment:
Does `num_indices()` equal to `index_entries_size()`?

If so, we don't need the field for `num_indices`.